code/world.dm

```diff
@@ -71,7 +71,8 @@ var/list/map_transition_config = MAP_TRANSITION_CONFIG
 	map_name = "Unknown"
 	#endif
 
-
+	for (var/typepath in (typesof(/datum/menu)-/datum/menu))
```

Comment:
basically, global list()s are initialized before global datum New()'s unless you put a datum/atom/etc's New before world's New(), then global datums and lists are initialized in the order they are defined as one step, rather than the normal two steps.